cypress/integration/unit_tests/manage_disaster_test.js

```diff
@@ -352,6 +443,66 @@ describe('Unit tests for manage_disaster.js', () => {
                       .get())
         .then((doc) => expect(doc.exists).to.be.false);
   });
+
+  /**
+   * Sets up fresh asset data (no feature collections), and prepares fake page
+   * with damage/map-bounds-related HTML elements.
+   * @return {Cypress.Chainable<Document>}
+   */
+  function setAssetDataAndCreateDamageInputsForScoreValidationTests() {
+    const currentData = createDisasterData(['NY']);
+    currentData['asset_data']['map_bounds_sw'] = '0, 1';
+    disasterData.set(getDisaster(), currentData);
+    return cy.document().then((doc) => {
+      // Lightly fake out jQuery so that we can use Cypress selectors. Might not
+      // work if manage_disaster.js starts doing fancier jQuery operations.
+      cy.stub(document, 'getElementById')
+          .callsFake((id) => doc.getElementById(id));
+      const damageSelect = doc.createElement('select');
+      damageSelect.id = 'damage-asset-select';
+      doc.body.appendChild(damageSelect);
+      const boundsDiv = doc.createElement('div');
+      boundsDiv.id = 'map-bounds-div';
+      doc.body.appendChild(boundsDiv);
+      const swInput = doc.createElement('input');
+      swInput.id = 'map-bounds-sw';
+      boundsDiv.appendChild(swInput);
+      const neInput = doc.createElement('input');
+      neInput.id = 'map-bounds-ne';
+      boundsDiv.appendChild(neInput);
+      return doc;
+    });
+  }
+
+  /**
+   * Sets up tests for score asset data validation beyond just damage.
+   * @return {Cypress.Chainable}
+   */
+  function setUpAssetValidationTests() {
+    // Clear out any modifications we've done to the document.
+    cy.visit('test_utils/empty.html');
+    return setAssetDataAndCreateDamageInputsForScoreValidationTests().then(
+        (doc) => {
+          cy.stub(document, 'createElement')
+              .callsFake((tag) => doc.createElement(tag));
+          const tbody = doc.createElement('tbody');
+          tbody.id = 'asset-selection-table-body';
+          doc.body.appendChild(tbody);
```

Comment:
createAndAppend a potentially useful operation (i think in import_test_utils)

Reply:
we need to append to doc.body, so would need more mocking. Good idea for future.